python/ray/serve/_private/constants.py

```diff
@@ -78,6 +83,39 @@
     600000,
 ]
 
+
+def _parse_latency_buckets(bucket_str: str) -> list:
+    if not bucket_str.strip():
+        return DEFAULT_LATENCY_BUCKET_MS
+    try:
+        # Convert string to list of floats
+        buckets = [float(x.strip()) for x in bucket_str.split(",")]
+        if not buckets:
+            raise ValueError("Empty bucket list")
+        if any(x <= 0 for x in buckets):
+            raise ValueError("Bucket values must be positive")
+        if sorted(set(buckets)) != buckets:
+            raise ValueError("Bucket values must be in ascending order and unique")
+        return buckets
+    except Exception as e:
+        raise ValueError(
+            f"Invalid format for {bucket_str}. "
+            f"Expected comma-separated positive numbers in ascending order. Error: {str(e)}"
+        )
```

Comment:
nit: put this into `utils.py` or another file to keep `constants.py` clear of code // easy to read

Reply:
I do like the appeal of clean constants file. I will move this to utils. I initially hesitating to do that because constants has wide scope, having it depend of utils increases risk of circular dependency in future.

Reply:
Yeah this makes sense. Perhaps a better approach would be to have a very small utils file that only contains the logic for parsing & defaulting environment variables. Could remove other boilerplate in this file that way too.